portal/app/apps/forms.py

```diff
@@ -21,19 +21,25 @@ class DeployNewApp(FlaskForm):
 
 
 class AppFileUpload(FlaskForm):
-    app_path = StringField("App path to save file at", validators=[InputRequired()])
+    app_path = StringField(
+        "App path of the form: home/filename", validators=[InputRequired()]
```

Comment:
Should not change the label of the field, use description parameter instead for this.
suggested: 
 ```
 app_path = StringField(
        "App path to save file at", description="absolute path (e. g. home/<filename> )", validators=[InputRequired()])
```